ivy/functional/backends/torch/__init__.py

```diff
@@ -16,6 +16,77 @@
 
 use = ivy.utils.backend.ContextManager(_module_in_memory)
 
+
```

Comment:
let's add a comment explaining the reason for doing this. Also, just to confirm the reverse operators work with backends other than `torch` right? Thanks for diving into this @yopknopixx 😄

Reply:
No problem @vedpatwardhan! I've added a short comment explaining what we're doing and why. Yes, it does indeed work with other frameworks. Torch and TF are done, I'll continue with Jax MXNet Numpy and Paddle shortly.